src/sidebar/components/NotebookView.js

```diff
@@ -133,7 +133,7 @@ function NotebookView({ loadAnnotationsService, streamer }) {
       <div className="NotebookView__filters">
         <NotebookFilters />
       </div>
-      <div className="NotebookView__results hyp-u-layout-row--align-center u-font--large">
+      <div className="NotebookView__results hyp-u-layout-row--align-center text-lg font-medium">
```

Comment:
The TW utility classes prefixed with `text-` and `font-` can be a little bit potentially confusing. In play are:

* `text-<color name>` - corresponding rule `color: <color-name>` 
* `text-<text size>` - corresponding rule `font-size: <text size>`
* `font-<weight name>`, e.g. `font-medium`, `font-normal`, `font-semibold` — sets a `font-weight` rule

Reply:
Thanks, this was a useful note.